app/models/concerns/avatarable.rb

```diff
@@ -12,10 +12,10 @@ module Avatarable
   def avatar_url
     return url_for(avatar.representation(resize: '250x250')) if avatar.attached? && avatar.representable?
 
-    if [SuperAdmin, User, Contact].include?(self.class) && email.present?
-      hash = Digest::MD5.hexdigest(email)
-      return "https://www.gravatar.com/avatar/#{hash}?d=404"
-    end
+    # if [SuperAdmin, User, Contact].include?(self.class) && email.present?
+    #   hash = Digest::MD5.hexdigest(email)
+    #   return "https://www.gravatar.com/avatar/#{hash}?d=404"
+    # end
```

Comment:
```suggestion
     if [SuperAdmin, User, Contact].include?(self.class) && email.present?
       hash = Digest::MD5.hexdigest(email)
       return "https://www.gravatar.com/avatar/#{hash}?d=404"
     end
```

let's keep this in since removing this will stop rendering gravatar for existing Contacts and Users. Let's take it up as a follow-up in https://github.com/chatwoot/chatwoot/issues/3853, with 
- ENV variable to control whether to use gravatar
- migration to populate the value for existing accounts 
- extend the contact avatar job to associate avatars to users on creation as well.

Reply:
There have been some concerns about using external services. But we could look into providing it as configurable options.